### DISCUSSION TITLE
Ошибка авторизации при входе в панель управления впервые

### DISCUSSION THREAD

**Discussion by rovius** [LacunaHub/.github#8] [Общее]
При попытке войти в панель управления с аккаунта, где раньше вход никогда не производился, появляется следующая ошибка:
![image](https://github.com/LacunaHub/.github/assets/121901513/e09f7e69-ff16-423e-9dff-f3fa423b4d1e)
URL страницы при этом `https://api.lacunabot.com/authorize/callback?code=<...>&state=<...>`
Если перезагрузить страницу, то произойдет перенаправление на стандартное "Авторизация не удалась", а если на главную страницу или профиль, то окажется, что вход всё же был выполнен.
Аккаунт до входа находился на сервере поддержки, в "Авторизованных приложениях" Лакуны не было.
Не критично, но неприятно.

**Comment by danliyev** [comment]
Исправлено в версии 4.11.3
